README.md

```diff
@@ -13,7 +13,7 @@ This project is licensed under the Apache-2.0 License.
 
 ## Building
 
-Kotlin Multiplatform projects are in [Alpha](https://kotlinlang.org/docs/reference/evolution/components-stability.html). The CRT interfaces are subject to change.
+CRT interfaces are subject to change.
 
 ### Linux/Unix
```

Comment:
**Question**: What's the updated procedure for building on Linux now? When building on my dev machine running AL2, I see the following error:

```
> Task :aws-crt-kotlin:cmakeBuildIosArm64 FAILED
cmake --build build/cmake-build/iosArm64 --config RelWithDebInfo -- -sdk iphoneos
GNU Make 3.82
Built for x86_64-koji-linux-gnu
Copyright (C) 2010  Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
Reading makefiles...
Reading makefile `Makefile'...
Updating makefiles....
 Considering target file `Makefile'.
  Looking for an implicit rule for `Makefile'.
  No implicit rule found for `Makefile'.
  Finished prerequisites of target file `Makefile'.
gmake: *** No rule to make target `iphoneos'.
 No need to remake target `Makefile'.
Updating goal targets....
Considering target file `iphoneos'.
 File `iphoneos' does not exist.
 Looking for an implicit rule for `iphoneos'.
 No implicit rule found for `iphoneos'.
 Finished prerequisites of target file `iphoneos'.
Must remake target `iphoneos'.
Failed to remake target file `iphoneos'.

FAILURE: Build failed with an exception.
```

Similar question for Windows.

Reply:
Yeah this is failing in CI for same reason. Kotlin plugin disables targets that can't build on a given host but it seems they don't disable the cinterop tasks even though that target won't be built. I imagine we'll just have to do the same and only enable the cmake tasks if a given target is enabled.